components/Templates/includes/functions-view_template.php

```diff
@@ -258,7 +258,7 @@ function frontier_do_subtemplate( $atts, $content ) {
 	$pod = pods( $atts[ 'pod' ], $atts[ 'id' ] );
 	$field_name = $atts[ 'field' ];
 
-	$entries = $pod->field( $field_name );
+	$entries = (array) $pod->field( $field_name );
```

Comment:
One of my fav things about this PR is  this commit

Reply:
But we ought to cast $entries as an array after we check if it's empty, right?

Reply:
I'm not sure if it matters but it doesn't hurt to change it.